schema/bom-1.5.schema.json

```diff
@@ -137,11 +137,39 @@
           "description": "The date and time (timestamp) when the BOM was created."
         },
         "tools": {
-          "type": "array",
-          "title": "Creation Tools",
-          "description": "The tool(s) used in the creation of the BOM.",
-          "additionalItems": false,
-          "items": {"$ref": "#/definitions/tool"}
+          "oneOf": [
+            {
+              "type": "object",
+              "title": "Creation Tools",
+              "description": "The tool(s) used in the creation of the BOM.",
+              "additionalProperties": false,
+              "properties": {
+                "components": {
+                  "type": "array",
+                  "additionalItems": false,
+                  "items": {"$ref": "#/definitions/component"},
+                  "uniqueItems": true,
+                  "title": "Components",
+                  "description": "A list of software and hardware components used as tools"
+                },
+                "services": {
+                  "type": "array",
+                  "additionalItems": false,
```

Comment:
Can the service list the underlying components used to generate the SBoM?

Reply:
The service can have an external reference to its corresponding SBOM